Float.Core/Net/RequestClient.cs

```diff
@@ -280,9 +298,17 @@ async Task<Response> Send(HttpRequestMessage request)
 
             try
             {
+                var originalUri = request.RequestUri?.AbsolutePath;
+
                 var httpResponse = await webClient.SendAsync(request).ConfigureAwait(false);
                 var responseContent = await httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
 
+                // Verify if a forbidden redirect happened.
+                if (httpResponse.StatusCode == HttpStatusCode.Forbidden && originalUri != request.RequestUri?.AbsolutePath)
```

Comment:
There is a potential safety issue here: if the request gets redirected to a different host, the `Authorization` header is still going to be included (possibly leaking credentials).

Reply:
I added a check for the host to only recreate the request if the host is the same